example/src/main/java/com/stripe/example/activity/CreateCardTokenActivity.kt

```diff
@@ -120,21 +122,16 @@ class CreateCardTokenActivity : AppCompatActivity() {
 
         fun createCardToken(cardParams: CardParams): LiveData<Token> {
             val data = MutableLiveData<Token>()
-
-            stripe.createCardToken(
-                cardParams,
-                callback = object : ApiResultCallback<Token> {
-                    override fun onSuccess(result: Token) {
-                        BackgroundTaskTracker.onStop()
-                        data.value = result
-                    }
-
-                    override fun onError(e: Exception) {
-                        BackgroundTaskTracker.onStop()
-                        Log.e("StripeExample", "Error while creating card token", e)
-                    }
+            viewModelScope.launch {
```

Comment:
Take a look at how I did this here:

https://github.com/stripe/stripe-android/blob/db1e174a1b4b9f4c3ead8f91dbb7b973f716108c/example/src/main/java/com/stripe/example/activity/CreateCardTokenActivity.kt#L117